tests/replacer/test_replacements_and_expiry.py

```diff
@@ -13,7 +14,7 @@
 
 
 @freeze_time("2024-5-13 09:00:00")
-class TestState:
+class TestState(unittest.TestCase):
```

Comment:
you don't need this

Reply:
Sorry, I left it in when trying to fix the mock error